docs/en/modules/tls.md

```diff
@@ -0,0 +1,285 @@
+# tls Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2016-07-04 | [PhoeniX](https://github.com/djphoenix) | [PhoeniX](https://github.com/djphoenix) | [tls.c](../../../app/modules/tls.c)|
```

Comment:
Please adjust the date to some day in December. Ideally it'd be the day we eventually merge the PR but we can't predict the future 😉 "Since" should give developers an indication when the module was add to the firmware in _our_ repository.